css/package.json

```
@@ -3,7 +3,9 @@
 	"version": "0.2.0",
 	"description": "Styles backing the Atlas Design System used by Microsoft's Developer Relations.",
 	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"lint": "stylelint ./src/**/*.scss",
```

It will fix as much as it can, nice for cleaning up after yourself when you're working through something but ignoring the linter 😂 

```suggestion
		"lint": "stylelint ./src/**/*.scss",
		"lint:fix": "npm run lint --fix",
```

---

👍 It is nice to have in package.json, because you can pass params to yarn, I was used to running `yarn lint --fix`, but this is nice.